### PR TITLE
[io/net] vlan.py: update the vlan test to use the latest utils

### PR DESCRIPTION
**Pull request by cforno12-zz**
Refactored and reorganized vlan test to use the latest avocado utils.

Signed-off-by: Cris Forno <fornosoccer12@gmail.com>

### CODE REVIEW

File: io/net/vlan_test.py [hunk 1]
```diff
@@ -175,32 +129,29 @@ def test_vlan_1_2230(self):
...
         time.sleep(5)
-        if self.ping_check_host(self.host_intf, self.ip_dic[self.peer_intf]):
-            self.fail("Ping test failed for vlan 1 & 2230 in host")
-        self.log.info("Ping test passed for vlan 1 % 2230 in host")
-        if self.ping_check_peer(self.peer_intf, self.ip_dic[self.host_intf]):
-            self.fail("Ping test failed for vlan 1 & 2230 in peer")
-        self.log.info("Ping test passed for vlan 1 % 2230 in peer")
+
+        self.host_interface.ping_check(peer_ip=self.peer_ip, count=5)
+        self.peer_interface.ping_check(peer_ip=self.peer_ip, count=5)
```
Comment (Naresh-ibm) [inline io/net/vlan_test.py:134]: is this something your doing self ping check?
Or ping from peer to host? if that is the case it should be,
self.peer_interface.ping_check(peer_ip=<host_ip>, count=5)

**Review by abdhaleegit — CHANGES_REQUESTED** [review]
@cforno12  Job logs please ?

File: io/net/vlan_test.py (hunk 1 above)
Comment (Naresh-ibm) [reply io/net/vlan_test.py:134]: @vaishnavibhat can you please re-check on this and attached the latest logs?
So that we can ask for next steps.

**Comment by abdhaleegit** [comment]
No activity from long time, Please reopen if you want to proceed further, also the PR does not work on latest switches as the cli is different, so not good idea to hardcode for one switch

**Pull request closed without merging**